### PR TITLE
support multiple email recipients

### PR DESCRIPTION
**Pull request by kumarabhishekpandey**
Support multiple email recipients
## Description

In current version of SE, email recipient `get_to_mail` is expected to be a single email address.
The change ensures that if `get_to_mail` contains multiple email addresses separated by commas, they are split into a list of individual email addresses before being passed to the sendmail function. This is necessary because sendmail expects a list of recipient addresses.
## Related Issue
[112](https://github.com/Nike-Inc/spark-expectations/issues/112)

## How Has This Been Tested?
Uploaded wheel file generated to Databricks cluster ran spark expectations job.

## Screenshots (if appropriate):
se_test_cases

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: spark_expectations/notifications/plugins/email.py [hunk 1]
```diff
@@ -50,7 +50,9 @@ def send_notification(
                 )
                 server.starttls()
                 text = msg.as_string()
-                server.sendmail(_context.get_mail_from, _context.get_to_mail, text)
+                server.sendmail(
+                    _context.get_mail_from, _context.get_to_mail.split(","), text
```
Comment (asingamaneni) [inline spark_expectations/notifications/plugins/email.py]: can you also do strip() after split for each email, so that we don't have spaces.

Comment (iamabhishek1995) [reply spark_expectations/notifications/plugins/email.py]: made the suggested changes...

Comment (asingamaneni) [reply spark_expectations/notifications/plugins/email.py]: Thank you!

**Review by asingamaneni — APPROVED** [review]
LGTM

**Pull request merged**